test_runner/regress/test_timeline_size.py

```diff
@@ -1062,7 +1062,7 @@ def eager_tenant_is_active():
 @pytest.mark.parametrize("activation_method", ["endpoint", "branch", "delete"])
 def test_lazy_attach_activation(neon_env_builder: NeonEnvBuilder, activation_method: str):
     # env.initial_tenant will take up this permit when attaching with lazy because of a failpoint activated after restart
-    neon_env_builder.pageserver_config_override = "concurrent_tenant_warmup = '1'"
+    neon_env_builder.pageserver_config_override = "concurrent_tenant_warmup = 1"
```

Comment:
we could post to the oss channel in discord about this.

Reply:
Will do

Reply:
https://discord.com/channels/1176467419317940276/1194680868439076874/1281238083987177532